### PR TITLE
Add new libnv-vulkan-producer lib for 470.63.01 release

### PR DESCRIPTION
**Pull request by h3o66**
required for the Wayland Vulkan WSI backend

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -1069,6 +1069,10 @@ nvidia-egl-wayland-tkg() {
     ln -s libnvidia-egl-wayland.so."${_eglwver}" "${pkgdir}"/usr/lib/libnvidia-egl-wayland.so.1
     ln -s libnvidia-egl-wayland.so.1 "${pkgdir}"/usr/lib/libnvidia-egl-wayland.so
 
+    if [[ $pkgver = 470.63* ]]; then
+	    install -Dm libnvidia-vulkan-producer.so."${pkgver}" "${pkgdir}"/usr/lib/libnvidia-vulkan-producer.so."${pkgver}"
```
Comment (Snoop05) [inline PKGBUILD:1073]:
```suggestion
      install -Dm755 libnvidia-vulkan-producer.so."${pkgver}" "${pkgdir}"/usr/lib/libnvidia-vulkan-producer.so."${pkgver}"
      ln -s libnvidia-vulkan-producer.so."${pkgver}" "${pkgdir}"/usr/lib/libnvidia-vulkan-producer.so.1
      ln -s libnvidia-vulkan-producer.so.1 "${pkgdir}"/usr/lib/libnvidia-vulkan-producer.so```

**Review by Snoop05 — CHANGES_REQUESTED** [review]
Should this be part of egl-wayland or another package?

**Comment by h3o66** [comment]
> Should this be part of egl-wayland or another package?

i was not sure about this, this is why i did add this to the wayland package.

**Comment by h3o66** [comment]
Seems like tkg already added this:
https://github.com/Frogging-Family/nvidia-all/commit/3e05cde408bfa8fce9a8fef0917c372eefbc2690
🐸

**Pull request closed without merging**